modules/cli/agentmanifest/agent.go

```diff
@@ -78,24 +77,6 @@ func insecurePing(host string) {
 	}
 }
 
-func testKubeConfig(kubeConfig, host string) error {
-	insecurePing(host)
-
-	restConfig, err := clientcmd.RESTConfigFromKubeConfig([]byte(kubeConfig))
-	if err != nil {
-		return fmt.Errorf("failed to test kubeconfig: %w", err)
-	}
-	client, err := kubernetes.NewForConfig(restConfig)
-	if err != nil {
-		return fmt.Errorf("failed to test build client from kubeconfig: %w", err)
-	}
-	_, err = client.Discovery().ServerVersion()
-	if err != nil {
-		return fmt.Errorf("failed to test connection to %s: %w", host, err)
-	}
-	return nil
-}
```

Comment:
**To reviewers:** in the search for kube client connection tests, this function was found and is unused by anything else.